contents/tutorials/performance-marketing.md

```diff
@@ -0,0 +1,117 @@
+
+![Performance marketing funnel](../images/tutorials/performance-marketing/funnel-basic.png)
+
+After creating this funnel, we can change this in a bunch of ways. First, we can break conversion down by our UTM properties to see conversion for different channels, sources, and more.
+
+![Breakdown funnel](../images/tutorials/performance-marketing/funnel-breakdown.png)
+
+Second, we can change the graph type to “Historical trends” to give us a better idea of how our performance marketing conversion is doing over time.
+
+![Historical trends type funnel](../images/tutorials/performance-marketing/funnel-type.png)
+
+On top of these, there are lots of settings to tweak your funnel such as “Conversion rate calculation,” “Conversion window limit,” and “Date range” to match the insights you want to get about your performance marketing. 
+
+## What are the benefits of measuring your performance marketing?
+
+As you’ve seen, PostHog has many tools for measuring performance marketing. This helps you figure out if it is making a difference in your business. Here are some recommendations on how you can use performance marketing measurements to make channels to improve your business.
+
+### Comparing channels
+
+Each of the ways of measuring performance in this tutorial allows you to breakdown by channel. This enables you to compare channels to find out which ones are working and which aren’t.
+
+You can compare traffic, signups, and conversion with your spend on each channel. Making sure the best channels get the most spend ensures efficiency. You can double down on ones that are working, and lower spend on ones that aren’t. If there are problem areas, you can quickly identify and fix them.
+
+Having a channel comparison also helps make future marketing strategy choices. A good performing channel encourages more investment into organic there or hiring people to help improve it. Knowing where you do best is key to creating a competitive advantage.
+
+### Test new channels, and improve existing ones
+
+Having measurements for performance marketing in place enables you to test new channels and improve existing ones. Without measurement, you won’t know how channels are doing. This leads to unnecessary waste and inaccurate decisions.
+
+Performance marketing trends are always changing. New channels are opening up. New ads are tested. What works now won’t always work. Measuring performance lets you analyze trends and adjust strategies based on what’s currently working.
+
+### Grow your business
+
+The ultimate goal of performance marketing is to help you make more money and grow your business. Measuring and tracking performance helps to ensure you’re on the right path toward achieving that goal.
```

Comment:
Like the intro, I feel like there's a lot of superfluous padding here. You could probably condense most of it down to a few bullet points.

Reply:
Simplified to a few bullet points.